src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -762,10 +742,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <Target Name="_ComputeUseBuildDependencyFile"
-          DependsOnTargets="_ComputeExcludeFromPublishPackageReferences;
+          DependsOnTargets="_ComputePackageReferencePublish;
                             _ParseTargetManifestFiles">
-
-    <!-- If there are no excluded packages or runtime store packages, and we are not linking, then we can reuse the deps file from the build, -->
+    <!-- If there are no runtime store packages, and we are not linking, then we can reuse the deps file from the build, -->
```

Comment:
Weird, when I submitted my review I guess I didn't commit this comment:

This updated comment seems now out of date with the logic of how `_UseBuildDependencyFile` is calculated as it is still predicated on there being no excluded from publish packages (if the set of package assets differ then it makes sense that the deps file can't be copied verbatim).